src/content-helper/editor-sidebar/title-suggestions/provider.ts

```diff
@@ -33,19 +35,29 @@ export class WriteTitleProvider {
 	/**
 	 * Returns a list of suggested titles for the given content.
 	 *
-	 * @param {string } content The content to generate titles for.
-	 * @param {number}  limit   The number of titles to return. Defaults to 3.
+	 * @param {string }     content The content to generate titles for.
+	 * @param {number}      limit   The number of titles to return. Defaults to 3.
+	 * @param {ToneProp}    tone    The tone to generate titles for.
+	 * @param {PersonaProp} persona The persona to generate titles for.
```

Comment:
Better update this to something like:

```suggestion
	 * @param {ToneProp}    tone    The tone to use when generating the titles.
	 * @param {PersonaProp} persona The persona to use when generating the titles.
```